internal/cmd/base/servers.go

```diff
@@ -575,7 +575,7 @@ func (b *Server) ConnectToDatabase(dialect string) error {
 				Colorful: false,        // Disable color
 			},
 		)
-		dbase = dbase.Session(&gorm.Session{Logger: newLogger})
+		dbase.DB = dbase.Session(&gorm.Session{Logger: newLogger})
```

Comment:
This is still causing `gorm` and `gorm/logger` to be imported here. I think that can be elimnated by moving this logger configuration into the `db` package. Probably by adding some `Options` to `db`, i.e.:

```go
opts := []db.Option
if os.Getenv("BOUNDARY_DISABLE_GORM_FORMATTER") == "" {
  opts = append(opts, db.WithGormFormatter(b.Logger))
}
dbase, err := db.Open(dbType, b.DatabaseUrl, opts...)
```

then in `db`:
```go
package db

// i dont think i have this exactly correct, but hopefully its close enough to get the idea
func Open(dbType DbType, connectionUrl string, options ...Option) (*DB, error) {
  // ...
  if opts.WithGormFormatter != nil {
    newLogger := logger.New(
      db.GetGormLogger(opts.WithGormFormatter),
      logger.Config{
        LogLevel: logger.Error,
        Colorful: false,
      },
    )
    db.DB = db.Session(&gorm.Session{Logger: newLogger})
  }

  // ...
}